views/forms/ConfigAssessmentFormFields/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { MenuItem } from '@material-ui/core'
+import ConfigurationCategoryCard from '../../components/ConfigurationCategoryCard'
+import TextInput from '../TextInput'
+import ControlledSelectInput from '../ControlledSelect'
+
+const ConfigAssessmentFormFields = ({
+  classes,
+  control,
+  colors,
+  index,
+  width,
+  onRemove,
+}) => {
+  return (
+    <ConfigurationCategoryCard
+      classes={classes}
+      formIndex={index}
+      onRemove={onRemove}
+      rowNum={index + 1}
+      width={width}
```

Comment:
CSS?

Reply:
The project uses a material library, the css are javascript objects, but we can't change these on the fly like that